src/bin/cargo/commands/test.rs

```diff
@@ -46,6 +46,7 @@ pub fn cli() -> App {
             "Exclude packages from the test",
         )
         .arg_jobs()
+        .arg_keep_going()
```

Comment:
Since `-j` and `--keep-going` are so closely related, perhaps this could be folded into the `arg_jobs` method?

Reply:
Done.